src/imports.js

```diff
@@ -4,6 +4,7 @@ import * as ReactDOM from 'react-dom';
 // API shared by Preact and React
 export const React = {
   createElement: _React_.createElement || _React_.h,
+  PropTypes: _React_.PropTypes || { oneOf: () => {} },
```

Comment:
figured it is the one function you use inside of the code. a comment would be nice.